tests/exclude_system_edl/host/host.c

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/host.h>
+#include <openenclave/internal/error.h>
+#include <openenclave/internal/tests.h>
+#include <stdio.h>
+
+#include "exclude_system_edl_u.h"
+
+int main(int argc, const char* argv[])
+{
+    oe_result_t result;
+    oe_enclave_t* enclave = NULL;
+
+    if (argc != 2)
+    {
+        fprintf(stderr, "Usage: %s ENCLAVE_PATH\n", argv[0]);
+        return 1;
+    }
+
+    const uint32_t flags = oe_get_create_flags();
+
+    if ((result = oe_create_exclude_system_edl_enclave(
+             argv[1], OE_ENCLAVE_TYPE_SGX, flags, NULL, 0, &enclave)) != OE_OK)
```

Comment:
Why TYPE_SGX instead of TYPE_AUTO?   This test doesn't appear to be SGX-specific

Reply:
You're right, I mostly just copied another test to quickly make this test. I'll fix it up to not be SGX-specific